static/showcases/blocked-segment.html

```diff
@@ -0,0 +1,117 @@
+    player.ready(() => {
+      // Create a component for displaying the blocked segment notification
+      const blockSegmentNotification = player.addChild('component', {
+        className: 'pbw-blocked-segment-notification'
+      });
+
+      // Hide the notification initially
+      blockSegmentNotification.hide();
+
+      // Variable to store the timeout ID for clearing later
+      let timeoutId = undefined;
+
+      // Listen for the 'loadeddata' event on the player, this ensures that text tracks are available
+      player.on('loadeddata', () => {
+        // Get the text track with the ID 'srgssr-blocked-segments'
+        const blockedSegments = player.textTracks().getTrackById('srgssr-blocked-segments');
+
+        // Add a listener for the 'cuechange' event on the blocked segments text track
+        blockedSegments?.on('cuechange', () => {
+          // Check if there are active cues in the blocked segments text track
+          if (blockedSegments.activeCues.length > 0) {
+            // If there is an existing timeout, clear it
+            if (timeoutId) clearTimeout(timeoutId);
+
+            // Parse the JSON content of the active cue to get the blocked segment information
+            const blockSegment = JSON.parse(blockedSegments.activeCues[0].text);
+            const blockReason = blockSegment.blockReason;
+
+            // Update the displayed notification text with the appropriate message based on the block reason
+            blockSegmentNotification.el().textContent = options.i18n[blockReason] ?? options.i18n['UNKNOWN'];
+
+            // Show the blocked segment notification
+            blockSegmentNotification.show();
+
+            // Set a timeout to hide the notification after a specified delay
+            timeoutId = setTimeout(() => {
+              blockSegmentNotification.hide();
+              timeoutId = undefined;
+            }, options.delay);
+          }
+        });
+      });
+    });
+  });
+
+  // Create a pillarbox player instance with the blockedSegmentNotification plugin
+  const player = pillarbox(
+    'video-element-id', {
+      fill: true,
+      plugins: {
+        blockedSegmentNotification: {
+          delay: 5000, // Delay in milliseconds before hiding the notification
+          i18n: { // Labels for different block reasons
+            STARTDATE: 'ⓘ A portion of the video is not yet available',
+            ENDDATE: 'ⓘ A section of the video has expired',
+            LEGAL: 'ⓘ For legal reasons, a section of the video content is omitted',
+            COMMERCIAL: 'ⓘ A commercial break has been excluded',
+            GEOBLOCK: 'ⓘ This segment of the video is not accessible in your region',
+            AGERATING18: 'ⓘ A segment suitable for 18 years and older is omitted',
+            AGERATING12: 'ⓘ A segment suitable for 12 years and older is omitted',
+            UNKNOWN: 'ⓘ Due to an unidentified issue, a segment of the video is omitted.'
```

Comment:
There is no need for that since we already have the translations.

Reply:
Removed.